lib/nerves_ssh.ex

```diff
@@ -50,16 +50,36 @@ defmodule NervesSSH do
     GenServer.call(__MODULE__, :info)
   end
 
+  @doc """
+  Add an SSH public key to the authorized keys
+
+  This will also attempt to save the key in `{USER_DIR}/authorized_keys`
+  """
+  @spec add_authorized_key(String.t()) :: :ok | {:error, :invalid_key}
```

Comment:
```suggestion
  @spec add_authorized_key(String.t()) :: :ok
```

I think that it's ok to just crash anyone who doesn't pass a string here.

Reply:
Fixed